compute/wps/views/auth.py

```diff
@@ -321,6 +332,10 @@ def login_mpc(request):
 
         host, port = g.groups()
 
+        from OpenSSL import SSL
```

Comment:
Any particular reason this import isnt at the top of the file?

Reply:
No particular reason.